docs/content/concepts/dagster-pipes/aws-ecs.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Integrating AWS ECS with Dagster Pipes | Dagster Docs"
+description: "Learn to integrate Dagster Pipes with AWS ECS to launch external code from Dagster assets."
+---
+
+# AWS ECS & Dagster Pipes
+
+This tutorial gives a short overview on how to use [Dagster Pipes](/concepts/dagster-pipes) with [AWS ECS](https://aws.amazon.com/ecs/).
+
+The [dagster-aws](/\_apidocs/libraries/dagster-aws) integration library provides the <PyObject object="PipesECSClient" module="dagster_aws.pipes" /> resource which can be used to launch AWS ECS tasks from Dagster assets and ops. Dagster can receive regular events like logs, asset checks, or asset materializations from jobs launched with this client. Using it requires minimal code changes on the task side.
+
+---
+
+## Prerequisites
+
+- **In the orchestration environment**, you'll need to:
+
+  - Install the following packages:
+
+    ```shell
+    pip install dagster dagster-webserver dagster-aws
+    ```
+
+    Refer to the [Dagster installation guide](/getting-started/install) for more info.
+
+  - **An existing boto3 client that can authenticate to AWS.** If you don't have this set up already, refer to the [boto3 quickstart](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html).
+
+- **In AWS**:
+
+  - An existing AWS account
+  - An AWS ECS task. To receive logs and events from a task container, it must have `"logDriver"` set to `"awslogs"` in `"logConfiguration"`.
+
+`dagster-pipes` can be called from multiple ECS containers at the same time. For the sake of this tutorial, we will be referring to only one container.
```

Comment:
I think we can remove this line, or at least move it elsewhere as it does not fit in "Prerequisites"